sway/tree/workspace.c

```diff
@@ -250,20 +250,35 @@ struct sway_container *workspace_by_name(const char *name) {
 		current_workspace = container_parent(focus, C_WORKSPACE);
 		current_output = container_parent(focus, C_OUTPUT);
 	}
-	if (strcmp(name, "prev") == 0) {
-		return workspace_prev(current_workspace);
-	} else if (strcmp(name, "prev_on_output") == 0) {
-		return workspace_output_prev(current_output);
-	} else if (strcmp(name, "next") == 0) {
-		return workspace_next(current_workspace);
-	} else if (strcmp(name, "next_on_output") == 0) {
-		return workspace_output_next(current_output);
-	} else if (strcmp(name, "current") == 0) {
-		return current_workspace;
+
+	char *name_cpy = strdup(name);
+	char *first_word = strtok(name_cpy, " ");
```

Comment:
What's the reasoning for getting and checking against the first word in this function?